scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java

```diff
@@ -122,17 +149,25 @@ public boolean doCheckAndSend() throws JobEmailNotificationException {
         }
     }
 
-    public void checkAndSendAsync() {
+    public void checkAndSendAsync(boolean withAttachment) {
         this.asyncMailSender.submit(() -> {
             try {
-                boolean sent = doCheckAndSend();
+                boolean sent = doCheckAndSend(withAttachment);
                 if (sent) {
                     jlogger.info(jobState.getId(), "sent notification email for finished job to " + getTo());
                 }
             } catch (JobEmailNotificationException e) {
                 jlogger.warn(jobState.getId(),
                              "failed to send email notification to " + e.getEmailTarget() + ": " + e.getMessage());
                 logger.trace("Stack trace:", e);
+            } catch (PermissionException e) {
```

Comment:
you don't need to explicitely catch each other exception, use here:

} catch (Exception e) {
  jlogger.warn(jobState.getId(), "failed to send email notification to " + e.getEmailTarget() + ": " + e.getMessage());
  logger.trace("Stack trace:", e);
}